include/antidote.hrl

```diff
@@ -185,7 +185,9 @@
 -type op() :: {op_name(), op_param()}.
 -type effect() :: term().
 
--type dcid() :: 'undefined' | {atom(),tuple()}. %% TODO, is this the only structure that is returned by riak_core_ring:cluster_name(Ring)?
+
+%% DC Id is the riak_core ring cluster name
+-type dcid() :: term().
```

Comment:
Was the old type not correct?

Reply:
This typing definition took a peek into the internals of `riak_core`. We do not know the type, we use it as a transparent `term` for our use cases.

Reply:
You can probably change it to:
```suggestion
-type dcid() :: undefined | riak_core_ring:riak_core_ring().
```